lib/public/AppFramework/Http/StreamGeneratorResponse.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+namespace OCP\AppFramework\Http;
+
+use Generator;
+use OCP\AppFramework\Http;
+
+/**
+ * @since 32.0.0
```

Comment:
```suggestion
 * @since 33.0.0
```
All @since are wrong here